frontend/src/app/alternatives/AlternativeForm.ts

```diff
@@ -0,0 +1,16 @@
+import { Alternative } from "../Hierarchy";
+
+export interface CreateAlternativeForm {
+    hierarchyId: string;
+    name: string;
+}
+
+export interface CreateAlternativeResponse {
```

Comment:
Maybe you could combine CreateAlternativeResponse and DeleteAlternativeForm to be one thing. Since you're using it in the service and the store, it should be something that makes sense anywhere. "HierarchyAlternative" or something is probably fine.